pkg/util/util.go

```diff
@@ -133,6 +133,17 @@ func HasKernelBootContainerImage(vmi *v1.VirtualMachineInstance) bool {
 	return true
 }
 
+func HasTPMDevice(vmiSpec *v1.VirtualMachineInstanceSpec) bool {
```

Comment:
Is possible to move this FN to a diffent package than util/util (https://go.dev/blog/package-names#bad-package-names)?

Reply:
if you don't want it in the utils we can move it to the staging/src/kubevirt.io/api/core/v1/types.go
Anyway, this seems like a good opportunity to fix this mess.

Reply:
> I suggest a separate PR to move all those functions to a "vmiproperty" package or similar.
That's a good idea. Do you have this PR? Would it be under staging/src/kubevirt.io/api/core/v1/vmiproperties.go?
just curious :)